### PR TITLE
German translation of Moneropedia entries starting with P

### PR DESCRIPTION
**Pull request by Wobole**
**German translation of the following Moneropedia entries:**

- Paper Wallet
- Payment ID
- Pedersen Commitment
- Pruning

### CODE REVIEW

File: _i18n/de/resources/moneropedia/paymentid.md [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
+### Kompakte Zahlungs-IDs und integrierte Adressen
 
-Since the 0.9 Hydrogen Helix version, Payment IDs can be encrypted and embedded in a payment address. The Payment IDs of this type should be 64-bits and are encrypted with a random one-time key known only to the sender and receiver.
+Seit der 0.9-Hydrogen-Helix-Version können Zahlungs-IDs verschlüsselt und in eine Zahlungsadresse eingebunden werden. Zahlungs-IDs dieser Art sollten eine Länge von 64 Bits haben; sie sind mit einem zufälligen Einmal-Schlüssel, der nur dem Sender und dem Empfänger bekannt ist, verschlüsselt.
 
-### Creating a Payment ID
-It is recommended to use the official wallet's `integrated_address` command to automatically generate Integrated Addresses that contain Compact Payment IDs. If you want to use the command line, you can generate Payment IDs as follows:
+### Erstellen einer Zahlungs-ID
+Es wird empfohlen, den `integrated_address`-Befehl des offiziellen Wallets zu verwenden, welcher automatisch integrierte Adressen generiert, die eine kompakte Zahlungs-ID beinhalten. Wenn du die Befehlszeile nutzen möchtest, kannst du Zahlungs-IDs wie folgt generieren:
```
Comment (stmax82) [inline _i18n/de/resources/moneropedia/paymentid.md:21]:
```suggestion
Es wird empfohlen, den `integrated_address`-Befehl des offiziellen Wallets zu verwenden, welcher automatisch integrierte Adressen generiert, die eine kompakte Zahlungs-ID enthalten. Wenn du die Befehlszeile nutzen möchtest, kannst du Zahlungs-IDs wie folgt generieren:
```

File: _i18n/de/resources/moneropedia/pedersen-commitment.md [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
 
-It is unclear which inputs are really being spent as the ring signature lists both the real inputs being spent and decoy inputs, therefore you don't actually know which input Pedersen commitments need to be summed. That's okay, because the @RingCT ring signature only has to prove that for one combination of the inputs the outputs are equal to the sum of the inputs. For mathematical reasons, this is impossible to forge.
+Welche Inputs tatsächlich gesendet werden ist unklar, da die Ringsignatur sowohl die in echt gesendeten Inputs als auch Input-Attrappen auflistet und demnach nicht genau klar ist, welche Inputs zusammengezählt werden müssen. Dies ist aber okay, weil die Ringsignatur der @Ring-CT lediglich nachweisen muss, dass bei einer bestimmten Kombination der Inputs die Outputs mit der Summe dieser Inputs übereinstimmen. Aus mathematschen Gründen ist es unmöglich, das Ergebnis zu fälschen.
```
Comment (stmax82) [inline _i18n/de/resources/moneropedia/pedersen-commitment.md:19]:
```suggestion
Welche Inputs tatsächlich ausgegeben werden ist unklar, da die Ringsignatur sowohl die in echt ausgegebenen Inputs, als auch Input-Attrappen auflistet, und demnach nicht genau klar ist, welche Inputs zusammengezählt werden müssen. Dies ist aber okay, weil die Ringsignatur der @Ring-CT lediglich nachweisen muss, dass bei einer bestimmten Kombination der Inputs die Outputs mit der Summe dieser Inputs übereinstimmen. Aus mathematschen Gründen ist es unmöglich, das Ergebnis zu fälschen.
```

File: _i18n/de/resources/moneropedia/pruning.md [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
-- *Create a new pruned blockchain from a full one:* A full blockchain that you already have will be used to create a new, pruned blockchain.
+- *Synchronisieren eines Pruned-Nodes von Grund auf:* Du synchronisierst und lädst lediglich 1/3 der gesamten Blockchain herunter.
+- *Pruning eines bestehenden Nodes:* Die bereits bestehende Blockchain wird gestutzt. Dies resultiert nicht in einer kleineren Blockchain, sondern wird Teile der Datei als leer markieren. Zukünftige Daten werden diesen nun freien Platz nutzen, damit die Datei nicht wächst, bis der Speicherplatz knapp wird.
+- *Erstellung einer neuen, gestutzten Blockchain aus einer kompletten Blockchain:* Eine komplette Blockchain, die du schon hast, wird zur Erstellung einer neuen, gestutzten Blockchain verwendet.
 
-Pruned nodes are very useful and preferable to @remote-nodes, but if possible, users should run a full node and opt for pruning only in case if necessary.
+Pruned-Nodes sind sehr nützlich und @Remote-Nodes vorzuziehen. Wenn es allerdings möglich ist, sollten Nutzer einen Full-Node betreiben und das Pruning nur im Falle der Notwendigkeit in Betracht ziehen.
```
Comment (stmax82) [inline _i18n/de/resources/moneropedia/pruning.md:15]:
```suggestion
Pruned-Nodes sind sehr nützlich und @Remote-Nodes vorzuziehen. Wenn es allerdings möglich ist, sollten Nutzer einen Full-Node betreiben und das Pruning nur im Falle der Notwendigkeit in Betracht ziehen.
```

Translation is fine, but I think the English version should explain why pruned nodes are "very useful" (=> improves own security) and why users should still opt to run a full node instead (=> helps sharing of the blockchain with other nodes).

File: _i18n/de/resources/moneropedia/paperwallet.md [hunk 4]
```diff
@@ -1,9 +1,9 @@
...
+summary: "Ein Paper-Wallet speichert die zum Senden und Empfangen von Monero notwendigen Informationen"
 ---
 
-{% include disclaimer.html translated="no" translationOutdated="no" %}
-### The Basics
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
+### Grundlagen
 
-A paper wallet stores the information necessary to send and receive Monero.
+Ein Paper-Wallet speichert die zum Senden und Empfangen von Monero notwendigen Informationen.
```
Comment (stmax82) [inline _i18n/de/resources/moneropedia/paperwallet.md:9]: Translation is fine, but the entry on paper wallets is way too short to be useful.

**Review by stmax82 — CHANGES_REQUESTED** [review]

File: _i18n/de/resources/moneropedia/pruning.md (hunk 3 above)
Comment (erciccione) [reply _i18n/de/resources/moneropedia/pruning.md:15]: Thanks for your suggestion. That's unrelated to this PR though, we can improve the original text later on.

**Review by erciccione — APPROVED** [review]

**Pull request merged**